lib/pages/people/view/person_avatar.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:cached_network_image/cached_network_image.dart';
+import 'package:flutter/material.dart';
+
+class PersonAvatar extends StatelessWidget {
```

Comment:
It's good to have a dedicated widget for this, but let's make it so we can reuse it in the multiple places where we need it. Use cases:
1) Your use case for the person avatar on web 
2) The person card on mobile (https://github.com/student-hub/acs-upb-mobile/blob/master/lib/pages/people/view/person_view.dart#L41-L53)
3) The profile card on the home page on mobile (https://github.com/student-hub/acs-upb-mobile/blob/master/lib/pages/home/profile_card.dart#L47-L56)

To do this, move this file to widgets/ and make `borderRadius` a widget param, then reuse this widget in the use cases above with the right radius.

Reply:
I moved it to widgets/ and added the param 'size' [here](https://github.com/student-hub/acs-upb-mobile/blob/396dff90559907416f9ea31238fb87d37c747e59/lib/widgets/person_avatar.dart)